src/mlpack/methods/decision_tree/decision_tree_impl.hpp

```diff
@@ -392,7 +392,7 @@ void DecisionTree<FitnessFunction,
                                       LabelsType&& labels,
                                       const size_t numClasses,
                                       const size_t minimumLeafSize,
-                                      const double minimumGainSplit)
+                                      const double /* minimumGainSplit */)
```

Comment:
Ah, I think this is actually a bug.  @Manthan-R-Sheth: should the `minimumGainSplit` parameter be passed to the `Train<>` call below?

Reply:
@rcurtin
Yes you are right, minimumGainSplit has to be passed toTrain<> call. I probably missed it. 
I will open a PR for the samw in 3 days as i am travelling now. Is it fine?

Reply:
Sure, that's fine---enjoy your traveling. :)

Reply:
Sure thanks.

Reply:
Sounds like a simple change to me, I guess if @ShikharJ and @Manthan-R-Sheth is fine with it we could incorporate the change in this PR. Let me know what you think.

Reply:
Sorry for the late response owing to my travelling.
Thanks for making the change. It looks good to go now.
 :D